### PR TITLE
Update net-standard-table.md add missing dot

### PR DESCRIPTION
**Pull request by Luis-Palacios**
Add missing dot for 4.61

# Title

Add missing dot for 4.61 in net-standard-table.md

## Summary

Add missing dot for 4.61 in net-standart-table.md

### CODE REVIEW

File: docs/includes/net-standard-table.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 |-----------------------------------|-------|--------|-------|-------|-------|--------|--------|-------|
 | .NET Core                         | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0    | 1.0    | 2.0   |
 | .NET Framework (with tooling 1.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2  | vNext  | 4.6.1 |
-| .NET Framework (with tooling 2.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.61   | 4.6.1  | 4.6.1 |
+| .NET Framework (with tooling 2.0) | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1   | 4.6.1  | 4.6.1 |
```
Comment (GuardRex) [inline docs/includes/net-standard-table.md]: Very tiny nit, but we're trying to maintain tables in clean format everywhere (and fix the bad ones as we go). Could you take out a space from the cell to get the column back in alignment?

**Review by guardrex — CHANGES_REQUESTED** [review]

File: docs/includes/net-standard-table.md (hunk 1 above)
Comment (LRPalacios) [inline docs/includes/net-standard-table.md]: Sure, just a sec

**Comment by GuardRex** [comment]
@LRPalacios Thanks for doing that and thanks for putting the fix in. We'll get it merged as soon as the build completes.

**Review by guardrex — APPROVED** [review]

**Comment by terrajobst** [comment]
LGTM

**Review by terrajobst — APPROVED** [review]

**Pull request merged**